specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,90 @@
+### SKUPriceID
+
+A unique identifier that defines the price of a SKU that incurred a charge.  This identifier represents a unique collection of nullable attributes that determine the price, as referenced on a price list published by a provider.
```

Comment:
```suggestion
A SKU Price ID is an identifier assigned to the unit price related information that’s associated with a charge. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU.
```

Reply:
Discussed on VS3 call and decided on following language:

> A SKU Price ID is a unique identifier that defines the price of a SKU that incurred a charge.  This identifier represents a unique collection of nullable attributes that determine the price.  SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU.